mobile/src/main/java/org/openhab/habdroid/ui/OpenHABWidgetAdapter.java

```diff
@@ -536,42 +538,27 @@ public void onStopTrackingTouch(SeekBar seekBar) {
     }
 
     public static class ImageViewHolder extends ViewHolder {
-        private final MySmartImageView mImageView;
-        private final View mParentView;
+        private final WidgetImageView mImageView;
         private int mRefreshRate;
 
         ImageViewHolder(LayoutInflater inflater, ViewGroup parent,
                 Connection conn, ColorMapper colorMapper) {
             super(inflater, parent, R.layout.openhabwidgetlist_imageitem, conn, colorMapper);
             mImageView = itemView.findViewById(R.id.imageimage);
-            mParentView = parent;
         }
 
         @Override
         public void bind(OpenHABWidget widget) {
-            // We scale the image at max 90% of the available height
-            mImageView.setMaxSize(mParentView.getWidth(), mParentView.getHeight() * 90 / 100);
```

Comment:
This seems to be lost, right?

Reply:
Nope, I just implemented somewhat more elegantly :-)
https://github.com/openhab/openhab-android/pull/850/files#diff-5974D32B68FE1032C17CD8F4CBDF981DR18

Reply:
I have a picuture that is has a larger height than before, this way it is higher than the available space.

Reply:
I assume I should be able to repro with any image that is taller than wide?

Reply:
I can't repro with a random portrait image I found via Google images (https://i.stack.imgur.com/YpDP4.png). It looks like this:
![screenshot_1524649682](https://user-images.githubusercontent.com/781373/39238490-a2c0fa8c-487e-11e8-863c-11a1154c190e.png)

Layout inspector says it's 1056x950, which perfectly fits the 90% ratio.

I guess I need more details :-)

Reply:
I created a picture with the same resolution:
![image](https://user-images.githubusercontent.com/22525368/39254617-c7899f64-48aa-11e8-8cd8-769b78b348c5.png)

Using this I should be able to see both green/blue strips, but I don't:
![screenshot_2018-04-25-17-08-35](https://user-images.githubusercontent.com/22525368/39254923-9882b90c-48ab-11e8-8c82-f411f45a5c4b.png)

Reply:
@maniac103 This issue is still open :)

Reply:
They are just scrolled out. I'm using a Fire Tablet 7 with a 7 inch display. Can you try the Nexus 7 (2012) emulator?

Reply:
Nexus 7 is 1280x800. I can create an emulator with 7" / 1024x600 though.

Reply:
Took me a while to understand the actual problem, but it's fixed now.

Reply:
Good you figured it out :+1: